vllm/worker/model_runner.py

```diff
@@ -683,6 +707,18 @@ def list_loras(self) -> Set[int]:
 
     @torch.inference_mode()
     def capture_model(self, kv_caches: List[KVCache]) -> None:
+        """Cuda graph capture a model.
+
+        Note that cuda graph performance gain is negligible if number
+        of batched tokens are less than 200. And since Cuda graph
```

Comment:
```suggestion
        Note that CUDA graph's performance gain is negligible if number
        of batched tokens are larger than 200. And since CUDA graph
```